sample-plugin/pom.xml

```diff
@@ -13,8 +13,8 @@
     <packaging>hpi</packaging>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <bom>2.303.x</bom>
-        <jenkins.version>2.303.1</jenkins.version>
+        <bom>weekly</bom>
+        <jenkins.version>2.314</jenkins.version>
```

Comment:
So how do you propose to update this? Via Dependabot or some other mechanism?

Reply:
Probably use @olblak ’s update cli

Reply:
yeah should be quite straightforward if it supports it, otherwise something like https://github.com/jenkinsci/helm-charts/blob/main/.github/workflows/sync-lts.yaml should work just fine too.

ideally fix dependabot, maybe worth looking briefly into it

Reply:
Unfortunately, updatecli doesn't support updating xml files at this stage. I started working on that a while ago and then unprioritized that.

Reply:
but we could easily implement a small shell script to handle that last operation similar to what we did with AMI configuration

Reply:
@olblak maybe simple text replacement would suffice?